### PR TITLE
MLT-0063 Use Ints for quantity

### PR DESCRIPTION
**Pull request by anotheroneofthese**
For our use-cases we do not need doubles for our Item models. This PR refactors this to use integers instead

### CODE REVIEW

File: src/main/kotlin/no/nb/mlt/wls/application/synqapi/synq/SynqBatchMoveItemPayload.kt [hunk 1]
```diff
@@ -127,11 +127,13 @@ data class Product(
     )
     val productVersionId: String,
     @Schema(
-        description = "Quantity of the product in the transport unit, uses float, however in our case we operate only in whole numbers.",
+        description =
+            "Quantity of the product in the transport unit. Accepts doubles, " +
+                "however, this is converted as we operate only in whole numbers.",
```
Comment (MormonJesus69420) [inline src/main/kotlin/no/nb/mlt/wls/application/synqapi/synq/SynqBatchMoveItemPayload.kt:132]:
```suggestion
        description = """Quantity of the product in the transport unit. 
            Accepts doubles, but they will be converted to integers.""",
```

File: src/main/kotlin/no/nb/mlt/wls/domain/WLSService.kt [hunk 2]
```diff
@@ -110,9 +110,9 @@ class WLSService(
                         "WLS DB has $itemsInStockQuantity stocked, and storage system tried to pick $pickedItemsQuantity"
                 }
             }
-            val quantity = Math.clamp(itemsInStockQuantity.minus(pickedItemsQuantity), 0.0, Double.MAX_VALUE)
+            val quantity = Math.clamp(itemsInStockQuantity.minus(pickedItemsQuantity).toLong(), 0, Int.MAX_VALUE)
```
Comment (MormonJesus69420) [inline src/main/kotlin/no/nb/mlt/wls/domain/WLSService.kt:113]: Shouldn't that be toInt instead of toLong?

**Review by MormonJesus69420 — CHANGES_REQUESTED** [review]

File: src/main/kotlin/no/nb/mlt/wls/domain/WLSService.kt (hunk 2 above)
Comment (anotheroneofthese) [reply src/main/kotlin/no/nb/mlt/wls/domain/WLSService.kt:113]: It is already an int, but the clamp function surprisingly does not take in an int.

**Review by MormonJesus69420 — APPROVED** [review]
LGTM

**Pull request merged**